script.skin.helper.service/default.py

```diff
@@ -99,36 +114,50 @@ def __init__(self):
                 if resultAction:
                     if "jsonrpc" in resultAction:
                         xbmc.executeJSONRPC(resultAction)
+                        xbmc.executeJSONRPC(resultAction)
```

Comment:
yes, it does return success indeed, even if the playback doesn't start.
Well, I will just remove that second command for now. If I receive some issues about it I can start looking at it again on some rainy day ;-)